src/lib.rs

```diff
@@ -47,64 +47,64 @@ pub fn example() {
     let mut y = 10;
 
     let label = Label::new()
-        .position(x, y)
-        .size(400, 16)
-        .text("Test Label")
-        .place(&mut window);
```

Comment:
Alright, I knew I should have done that. also, the indenting thing is really annoying. I don't like how it does that. I was working on a project earlier and it did that, aligned it upwith the double colon.I'll fix up this pull request tomorrow.